### PR TITLE
[PLAM-48] Password reset page

### PR DESCRIPTION
**Pull request by janicejl**
https://sig-gis.atlassian.net/browse/PLAM-48

Reset page:
image

With email filled in:
image

Forget password link:
image

### CODE REVIEW

**Comment by janicejl** [comment]
To add -- Once Reset Password button is sent. The reset password dialog box confirming that an email was sent added by Sterling will show up. 
Closing that dialog box brings the user back to /home.

File: src/interface/src/app/login/login.component.ts [hunk 1]
```diff
@@ -5,17 +5,12 @@ import { Router } from '@angular/router';
 import { AuthService } from '../services';
 import { MatDialog } from '@angular/material/dialog';
```
Comment (macieksmuga) [inline src/interface/src/app/login/login.component.ts:6]: I don't think there's any more dialog in this component, is there? This and other references to MatDialog can probably be safely removed from this and the associated test file

File: src/interface/src/app/login/login.component.spec.ts [hunk 2]
```diff
@@ -2,20 +2,19 @@ import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { Router } from '@angular/router';
+import { By } from '@angular/platform-browser';
+import { MatDialog, MatDialogModule } from '@angular/material/dialog';
```
Comment (macieksmuga) [inline src/interface/src/app/login/login.component.spec.ts:6]: Safe to remove this and other MatDialog references here

File: src/interface/src/app/home/about/about.component.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-planscape-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.scss'],
+})
+export class AboutComponent {
+  readonly text1: string = `
```
Comment (macieksmuga) [inline src/interface/src/app/home/about/about.component.ts:9]: Why not have this in the template verbatim?

**Review by macieksmuga — COMMENTED** [review]
Thanks so much for doing this! Looks great, ust a couple of minor comments.

File: src/interface/src/app/forget-password/forget-password.component.scss [hunk 4]
```diff
@@ -0,0 +1,90 @@
+.forget-password-root {
```
Comment (pflopez) [inline src/interface/src/app/forget-password/forget-password.component.scss:1]: you can avoid creating this extra element and just use `:host` to style the component

File: src/interface/src/app/home/about/about.component.ts (hunk 3 above)
Comment (janicejl) [reply src/interface/src/app/home/about/about.component.ts]: Done.

File: src/interface/src/app/login/login.component.spec.ts (hunk 2 above)
Comment (janicejl) [reply src/interface/src/app/login/login.component.spec.ts]: Done.

File: src/interface/src/app/login/login.component.ts (hunk 1 above)
Comment (janicejl) [reply src/interface/src/app/login/login.component.ts]: Done.

File: src/interface/src/app/forget-password/forget-password.component.scss (hunk 4 above)
Comment (janicejl) [reply src/interface/src/app/forget-password/forget-password.component.scss:1]: It wasn't working for me when I switched this to `:host`. The flex display is not applied.

**Review by janicejl — COMMENTED** [review]
Thanks for the review.

File: src/interface/src/app/forget-password/forget-password.component.scss (hunk 4 above)
Comment (macieksmuga) [reply src/interface/src/app/forget-password/forget-password.component.scss:1]: This is OK by me, thanks for checking!

**Review by macieksmuga — APPROVED** [review]

**Pull request merged**